README.md

```diff
@@ -80,20 +88,49 @@ where
 - `api.test.identity.domain`: the domain associated with a Cloud Identity customer. Temporary users
   will be created on this domain during the execution of the test.
 
+- `api.test.adminUserId`: Email ID of admin account.
+
+- `api.test.rootUrl`: the base url of api endpoint. For prod the `rootUrl` is https://cloudsearch.googleapis.com/. For staging the `rootUrl` is https://staging-cloudsearch.sandbox.googleapis.com/.
+
+- `api.test.authInfoUser1`: Authentication information for User1. This consists of the email id of User1, the path to the folder containing the user's Stored Credentials and the path to the JSON file containing client secret for desktop app.\
+  NOTE: The Stored Credential of the user must be kept in a file called StoredCredential, and we only need to pass the path of the folder containing the file.
+
+- `api.test.authInfoUser2`: Authentication information for User2. This consists of the email id of User2, the path to the folder containing the user's Stored Credentials and the path to the JSON file containing client secret for desktop app.\
+  NOTE: The Stored Credential of the user must be kept in a file called StoredCredential, and we only need to pass the path of the folder containing the file.
+
+- `api.test.authInfoUser3`: Authentication information for User3. This consists of the email id of User3, the path to the folder containing the user's Stored Credentials and the path to the JSON file containing client secret for desktop app.\
+  NOTE: The Stored Credential of the user must be kept in a file called StoredCredential, and we only need to pass the path of the folder containing the file.
+
+- `api.test.authInfo`: Authentication information for Admin. This consists of the email id of Admin, the path to the folder containing the admin's Stored Credentials and the path to the JSON file containing client secret for desktop app.\
+  NOTE: The Stored Credential of the admin must be kept in a file called StoredCredential, and we only need to pass the path of the folder containing the file.
+
+- `api.test.groupPrefix` : The name of a group created for testing purpose. This test must contain User3 and should not contain User1 and User2.
```

Comment:
Modify the comment to "This group must contain User3 and should not contain User1 and User2."

Reply:
Done. Thanks for pointing out.